zanata-war/src/main/java/org/zanata/async/AsyncTaskManager.java

```diff
@@ -101,7 +101,7 @@ public void execute() {
                     taskFuture.set(returnValue);
                 } catch (Throwable t) {
                     taskFuture.setException(t);
-                    log.warn(
+                    log.error(
                             "Exception when executing an asynchronous task.", t);
```

Comment:
Perhaps we should log the stacktrace as WARN, and log the exception.toString as ERROR.